### ISSUE TITLE
Workspace Clean Hangs (either on startup or a forced clean)

### ISSUE THREAD

**Issue by kjq** [redhat-developer/vscode-java#2064]
When working with either a single module or multi-module project VSCode will stall when "rebuilding the workspace" during startup or forced.  Usually, this will stop at a specific point in the project.

The project is a straight-forward Quarkus project with Lombok and Mapstruct (annotation processing).  When VSCode does work it works perfectly fine.

For whatever reason, this file `org.eclipse.jdt.apt.core.prefs` does not get created or gets created partially and copying the below properties into every `.settings` folder gets past it.

```
eclipse.preferences.version=1
org.eclipse.jdt.apt.aptEnabled=true
org.eclipse.jdt.apt.genSrcDir=target/generated-sources/annotations
org.eclipse.jdt.apt.genTestSrcDir=target/generated-test-sources/test-annotations
```

It is these lines that are missing:
**org.eclipse.jdt.apt.genSrcDir=target/generated-sources/annotations
org.eclipse.jdt.apt.genTestSrcDir=target/generated-test-sources/test-annotations**

##### Environment
- Operating System: WSL2
- JDK version: 15 (vscode) / 11 project
- Visual Studio Code version: 1.59.0

##### Steps To Reproduce
1. Open a single module or multi-module project
2. Remove the file above if it exists and see it stall or it just stalls naturally (cleaning the workspace or startup)
3. Copy the file above into the .settings folder
4. Close VSCode
5. Restart 

It will either get past initial stalling point and stop at another folder (if not copied into every folder) or succeed.

##### Current Result

Startup or cleaning the workspace hangs forever.

##### Expected Result

Workspace fully comes up.

##### Additional Informations

Logs provide no real details as to what is happening.  The solution came about trying to diff a folder prior to stalling with the one that stalled.  Sometimes the stall point is a false-positive and it a prior folder needs the update.

Lost many hours on this when it happens.

**Comment by kjq** [comment]
Maybe related/fixed by https://github.com/redhat-developer/vscode-java/issues/2020

**Comment by fbricon** [comment]
Please check with the newly released vscode-java 0.81.0

**Comment by kjq** [comment]
Have not had this in a while.
